src/Experiment.jsx

```diff
@@ -12,7 +13,11 @@ emitter.addActiveVariantListener(function (
   if (skipSave) {
     return;
   }
-  store.setItem('PUSHTELL-' + experimentName, variantName);
+  if (emitter.withCookie()) {
```

Comment:
Hey this looks great, thanks for submitting this PR!

Justing thinking: I'm looking at https://caniuse.com/?search=localstorage
and it seems that `window.localStorage` is supported by almost any modern browser (96.6% of the users at the moment).

So maybe rather than manually setting this option via `emitter` we could extend the existing `store` to fallback to cookies when `localStorage` is not supported.


For example something like:

```.js
import Cookies from 'js-cookies';

// https://github.com/Modernizr/Modernizr/blob/d4c7b6082709e32fb0589ba38aa96581d44ce395/feature-detects/storage/cookies.js#L16-L35
const cookiesEnabled = (() => {
  try {
    document.cookie = 'cookietest=1';
    const ret = document.cookie.indexOf('cookietest=') !== -1;
    document.cookie = 'cookietest=1; expires=Thu, 01-Jan-1970 00:00:01 GMT';
    return ret;
  } catch (e) {
    return false;
  }
})();

// https://github.com/Modernizr/Modernizr/blob/d4c7b6082709e32fb0589ba38aa96581d44ce395/feature-detects/storage/localstorage.js#L17-L46
const localStorageEnabled = (() => {
  const mod = 'modernizr';
  try {
    localStorage.setItem(mod, mod);
    localStorage.removeItem(mod);
    return true;
  } catch (e) {
    return false;
  }
})();

const store = {
    getItem(key) {
        if (localStorageEnabled) {
            return localStorage.getItem(key);
        }
        if (cookiesEnabled) {
            return Cookies.get(key);
        }
    }, 
    setItem(key, value) {
        if (localStorageEnabled) {
            return localStorage.setItem(key, value);
        }
        if (cookiesEnabled) {
            return Cookies.set(key, value);
        }
    }
};

export default store;
```

could replace the existing `store` and be more resilient? 🤔 
This includes some snippets from Modernizr to perform feature detection and use `cookie-js` to read/write cookies. The library is only 800 bytes, so it should be fine to add it as dependency. 
What do you think?

Reply:
Hey, 
Thanks for the comment! You are right, this might be the correct way of implementing the feature. Also, I believe that the code you provided is nicely written and elegant. Have you tested the code above with the tool? If you haven't, I have no problem of doing it and provide any changes needed!

Reply:
Hey! 
I added the provided code, reverted my previous commit and installed js-cookie as well. The code tested on both of my dummy react projects, everything worked as expected, even when i blocked local storage the code created a cookie as expected. For any change/fix/test/whatever please let me know.

Reply:
Hey @b10z sorry for the late reply, unfortunately I didn't have much free time recently, but I'll have a look next week. Thanks again for contributing to the project!